helm-charts/sawmills-collector/tests/backend_drain_test.yaml

```
@@ -0,0 +1,168 @@
+
+  - it: should preserve explicit main collector service extensions when configured inline
+    template: backend-drain-configmap.yaml
+    values:
+      - ../values.yaml
+      - fixtures/backend_drain_inline_extensions.yaml
+    set:
+      loadBalancer.enabled: true
+    asserts:
+      - matchRegex:
+          path: data["config.yaml"]
+          pattern: '(?ms)^service:\n\s+extensions:\n\s+- health_check\n\s+- basicauth/client\n\s+- backend_drain\n'
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)